### PR TITLE
Person finder capability

### PR DESCRIPTION
**Pull request by cuong-epi**

### CODE REVIEW

File: src/Foundation.Find.Cms/Models/Pages/PersonItemPage.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    [ImageUrl("~/assets/icons/cms/blocks/contact.png")]
+    public class PersonItemPage : FoundationPageData
+    {
+        [CultureSpecific]
+        [Display(Name = "Job Title", GroupName = SystemTabNames.Content, Order = 1)]
+        public virtual string JobTitle { get; set; }
+
+        [CultureSpecific]
+        [Display(GroupName = SystemTabNames.Content, Order = 2)]
+        public virtual string Location { get; set; }
```
Comment (ngocnguyen-epi) [inline src/Foundation.Find.Cms/Models/Pages/PersonItemPage.cs:24]: Use SelectionFactory and add extend the settings to add a list of locations and sectors on Home page

File: src/Foundation.Find.Cms/Models/Pages/PersonItemPage.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+        [CultureSpecific]
+        [Display(GroupName = SystemTabNames.Content, Order = 3)]
+        public virtual string Sector { get; set; }
```
Comment (ngocnguyen-epi) [inline src/Foundation.Find.Cms/Models/Pages/PersonItemPage.cs:28]: Use SelectionFactory and add extend the settings to add a list of locations and sectors on Home page

File: src/Foundation.Find.Cms/Models/Pages/PersonItemPage.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+using EPiServer.DataAbstraction;
+using EPiServer.DataAnnotations;
+using EPiServer.Web;
+using Foundation.Cms.Pages;
+
+namespace Foundation.Find.Cms.Models.Pages
+{
+    [ContentType(DisplayName = "Person Item Page", 
+        GUID = "b5af511b-96c9-4ad7-828f-254924542430", 
+        Description = "Person Item Page shows info of specific person",
```
Comment (ngocnguyen-epi) [inline src/Foundation.Find.Cms/Models/Pages/PersonItemPage.cs:12]: Please follow standard: Capitalisation (first letter in uppercase and the remaining letters in lowercase)

File: src/Foundation.Find.Cms/Models/Pages/PersonItemPage.cs [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        GroupName = FindTabNames.Person)]
+    [AvailableContentTypes(Availability.Specific, Exclude = new[] { typeof(PageData) })]
+    [ImageUrl("~/assets/icons/cms/blocks/contact.png")]
+    public class PersonItemPage : FoundationPageData
+    {
+        [CultureSpecific]
+        [Display(Name = "Job Title", GroupName = SystemTabNames.Content, Order = 1)]
```
Comment (ngocnguyen-epi) [inline src/Foundation.Find.Cms/Models/Pages/PersonItemPage.cs:19]: Please follow standard: Capitalisation (first letter in uppercase and the remaining letters in lowercase)
Update "Order" to 10-20-.....

File: src/Foundation.Find.Cms/Models/Pages/PersonItemPage.cs [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+        [Display(GroupName = SystemTabNames.Content, Order = 4)]
+        public virtual string Tel { get; set; }
+
+        [CultureSpecific]
+        [Display(GroupName = SystemTabNames.Content, Order = 5)]
+        public virtual string Email { get; set; }
+
+        [UIHint(UIHint.Image)]
+        [Display(Name = "Person Image", GroupName = SystemTabNames.Content, Order = 6)]
```
Comment (ngocnguyen-epi) [inline src/Foundation.Find.Cms/Models/Pages/PersonItemPage.cs:38]: Please follow standard: Capitalisation (first letter in uppercase and the remaining letters in lowercase)

File: src/Foundation.Find.Cms/Models/Pages/PersonListPage.cs [hunk 6]
```diff
@@ -0,0 +1,16 @@
+﻿using EPiServer.DataAbstraction;
+using EPiServer.DataAnnotations;
+using Foundation.Cms.Pages;
+
+namespace Foundation.Find.Cms.Models.Pages
+{
+    [ContentType(DisplayName = "Person List Page", 
+        GUID = "4f0203b6-d49e-4683-9ce6-ede8c37c77d3", 
+        Description = "Person List Page helps users can find people within an organisation",
```
Comment (ngocnguyen-epi) [inline src/Foundation.Find.Cms/Models/Pages/PersonListPage.cs:9]: Please follow standard: Capitalisation (first letter in uppercase and the remaining letters in lowercase)

File: src/Foundation.Find.Cms/Models/Pages/PersonItemPage.cs [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+        [CultureSpecific]
+        [Display(GroupName = SystemTabNames.Content, Order = 2)]
+        public virtual string Location { get; set; }
+
+        [CultureSpecific]
+        [Display(GroupName = SystemTabNames.Content, Order = 3)]
+        public virtual string Sector { get; set; }
+
+        [Display(GroupName = SystemTabNames.Content, Order = 4)]
+        public virtual string Tel { get; set; }
```
Comment (ngocnguyen-epi) [inline src/Foundation.Find.Cms/Models/Pages/PersonItemPage.cs:31]: Change to "Phone"

File: src/Foundation.Find.Cms/Models/Pages/PersonItemPage.cs [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+        public virtual string Email { get; set; }
+
+        [UIHint(UIHint.Image)]
+        [Display(Name = "Person Image", GroupName = SystemTabNames.Content, Order = 6)]
+        public virtual ContentReference Image { get; set; }
+
+        [CultureSpecific]
+        [UIHint(UIHint.Textarea)]
+        [Display(GroupName = SystemTabNames.Content, Order = 7)]
+        public virtual string About { get; set; }
```
Comment (ngocnguyen-epi) [inline src/Foundation.Find.Cms/Models/Pages/PersonItemPage.cs:44]: Use RTE (XhtmlString)

File: src/Foundation/Features/Persons/PersonListPage/Index.cshtml [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+@section AdditionalScripts
+{
+    <script type="text/javascript" src="/Assets/js/vendors/jsuri/jsuri-1.1.1.min.js"></script>
+    <script type="text/javascript" src="/Assets/js/vendors/bootstrap-slider/bootstrap-slider.js"></script>
+    <script type="text/javascript" src="/Assets/js/Features/Selective/Persons.js"></script>
+    <script type="text/javascript">
+        var persons = new Persons();
+        function GetMap() { persons.Init(); };
+    </script>
+    <script type='text/javascript' src='https://www.bing.com/api/maps/mapcontrol?&callback=GetMap' async defer></script>
```
Comment (ngocnguyen-epi) [inline src/Foundation/Features/Persons/PersonListPage/Index.cshtml:15]: Remove unused js

File: src/Foundation/Features/Persons/PersonListPage/Index.cshtml [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+@model PersonListViewModel
+
+@section AdditionalScripts
+{
+    <script type="text/javascript" src="/Assets/js/vendors/jsuri/jsuri-1.1.1.min.js"></script>
+    <script type="text/javascript" src="/Assets/js/vendors/bootstrap-slider/bootstrap-slider.js"></script>
+    <script type="text/javascript" src="/Assets/js/Features/Selective/Persons.js"></script>
+    <script type="text/javascript">
+        var persons = new Persons();
+        function GetMap() { persons.Init(); };
```
Comment (ngocnguyen-epi) [inline src/Foundation/Features/Persons/PersonListPage/Index.cshtml:13]: GetMap() ???

File: src/Foundation/Features/Persons/PersonListPage/Index.cshtml [hunk 11]
```diff
@@ -0,0 +1,122 @@
+﻿@using EPiServer.Web.Mvc.Html
+@using EPiServer.Find
+@using Foundation.Find.Cms.Persons.ViewModels
+@model PersonListViewModel
+
+@section AdditionalScripts
+{
+    <script type="text/javascript" src="/Assets/js/vendors/jsuri/jsuri-1.1.1.min.js"></script>
+    <script type="text/javascript" src="/Assets/js/vendors/bootstrap-slider/bootstrap-slider.js"></script>
```
Comment (ngocnguyen-epi) [inline src/Foundation/Features/Persons/PersonListPage/Index.cshtml:9]: Remove unused js

**Comment by davidknipe** [comment]
Can we change the folder name from Persons to People? It reads a little easier saying People

**Comment by lunchin** [comment]
was weird issues i think in the name of the branch starting with #

**Pull request closed without merging**